tests/fixtures/git/github.com/demo/demo/demo.egg-info/PKG-INFO

```diff
@@ -6,9 +6,5 @@ Home-page: https://github.com/demo/demo
 Author: Sébastien Eustace
 Author-email: sebastien@eustace.io
 License: MIT
-Platform: UNKNOWN
```

Comment:
This change is from a known unstable test and should be dropped.

Reply:
reverted this change